api/controllers/calendarEntry.controller.ts

```diff
@@ -64,11 +64,25 @@ export const seedDatabaseWithEntry = async (
   return;
 };
 
+export const createCalendarEntry = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const entry = await CalendarEntry.create(req.body);
+  res.status(201);
+  res.json(entry);
```

Comment:
You should be able to chain these, i.e. `res.status(201).json(entry)`

Reply:
Also looks like we're choosing to return the body here with the 201 status, which works for me 👍 turns out that's actually correct. If we were weren't returning a body, it'd be more appropriate to return a 204 status

Reply:
More info: https://httpwg.org/specs/rfc7231.html#status.201